src/sections/Community/community.style.js

```diff
@@ -260,6 +260,115 @@ const CommunitySectionWrapper = styled.div`
         }
     }
 
+    .recognition-program {
```

Comment:
why defining styles again ? is there no styles to reuse from this page?

Reply:
Updated the styles. Removed redundant style definitions and used similar style definitions from other sections. Only styles related to badge alignment and responsiveness is there as it is specific to this section.

Reply:
@souvikinator i was mentioning to use reuse the styles which means the classnames if the other section have the same structure so in that way you don't  have to write styles again. I am sure in this page only there might be jumbotrons so see if you can use there classes instead writing again.

Reply:
@sudhanshutech  in the current implementation, I'm already reusing styles from the `meshmate` section (hence the `.meshmate` class addition) for the section layout, margins, text content and responsive behavior. 

The styles you see in the code are specifically for the **badge stack visualization** and its responsive behavior since this is a unique visual element that is not present in any section.

Are you suggesting we come up with section design/UI that sticks more closely to the existing classes so that we don't need to define the badge related or in fact any extra style? That would mean changing the section UI.